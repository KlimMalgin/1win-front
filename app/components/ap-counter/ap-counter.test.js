/* eslint-env jest */
import { mount, createLocalVue } from 'vue-test-utils';
import apCounter from './ap-counter.vue';
import Vue from 'vue';
import Vuex from 'vuex';

// import sinon from 'sinon';

Vue.use( Vuex );

const createStore = () => new Vuex.Store( { getters: () => 0 } );

describe( 'ap-counter.vue', () => {
    let localVue;
    let wrapper;

    beforeAll( () => {
        localVue = createLocalVue();
        wrapper = mount( apCounter, {
            localVue,
            store : createStore(),
        } );
    } );

    it( 'snapshot. Проверяем соответствие шаблону', () => {
        expect( wrapper.element ).toMatchSnapshot();
    } );

} );
