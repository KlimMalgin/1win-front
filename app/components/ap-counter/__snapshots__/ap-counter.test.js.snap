// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ap-counter.vue snapshot. Проверяем соответствие шаблону 1`] = `
<div
  class="ap-counter"
>
  <div>
    <span>
      Value: 
    </span>
     
    <span>
      
    </span>
  </div>
   
  <button>
    Increment
  </button>
</div>
`;
